.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,37 @@
+name: "Feature request"
+description: "You have a neat idea that should be implemented?"
+labels: ["enhancement", "0. to triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### 👍Thanks for reporting issues back to Nextcloud!
```

Comment:
Add a space after `👍`.

Github doesn't seem to let me put more than one suggestion?

Reply:
> Add a space after `👍`.
> 
> Github doesn't seem to let me put more than one suggestion?

write it as a comment. Just so you know, i only copy paste what we have on the old one, so give as many suggestions as you want :)